airbyte-config/init/src/main/resources/config/STANDARD_SOURCE/778daa7c-feaf-4db6-96f3-70fd645acc77.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Shouldn't this be in `STANDARD_SOURCE_DEFINITION` and not `STANDARD_SOURCE`?

Reply:
I am not sure i understand your comment...
I don't see folders named `STANDARD_SOURCE_DEFINITION` in `airbyte-config/init/src/main/resources/config/` ?

Where is the `STANDARD_SOURCE_DEFINITION` ?